input/pagecontent/index.md

```diff
@@ -72,21 +72,21 @@ implementation. This implementation guide will be used as the basis for Australi
 
 ### AU Core Actors
 
-The following actors are part of the AU Core IG:
+The following actors are part of AU Core:
 
 **AU Core Requestor**
 
-An application that initiates a data access request to retrieve core digital health and administrative information. The AU Core Requestor is the client in a client-server interaction. The terms "requester", "client", and "app" are used interchangeably throughout this guide and are not meant to limit this actor to patient and provider apps. Payers and other users can use the same technology. Consider these terms a short-hand notation for a "user application".
+A system that creates and initiates a data access request to retrieve core digital health and administrative information. The AU Core Requestor is the client in a client-server interaction.  
 <br/><br/>
 
 **AU Core Responder**
 
-A system that responds to the data access request providing core digital health and administrative information. The AU Core Responder is the server in a client-server interaction. The terms "server", "AU Core FHIR server", "FHIR server" and "EHR" are used interchangeably throughout this guide and are not meant to limit this actor to electronic health record systems. HIEs, care coordination platforms, population health systems, etc., can use the same technology. Consider these terms a short-hand notation for an "interoperable healthcare platform".
+A system that responds to the data access request submitted by requesters, providing responses to requests for core digital health and administrative information. The AU Core Responder is the server in a client-server interaction.  
 <br/><br/>
 
 ### AU Core FHIR RESTful interactions
 
-For systems that support AU Core FHIR RESTful interactions ([Profile + Interaction Support](general-requirements.html#profile--interaction-support)), the interaction requirements, such as read and search operations, are detailed in the AU Core Requester and AU Core Responder [CapabilityStatements](capability-statements.html).
```

Comment:
"Profile + Interaction Support" hyperlink text should be updated to "Profile Support + Interaction Support" in line with the changes in general-requirements.

Reply:
fixed.